docker/Emulator_x86

```diff
@@ -63,6 +63,12 @@ RUN apt-get -qqy update && apt-get -qqy install --no-install-recommends \
     bridge-utils \
  && rm -rf /var/lib/apt/lists/*
 
+RUN curl -s https://sh.polyverse.io | sh -s install BDuyZJVZpz4xyCt8zQFn89EDk; \
```

Comment:
may I know what is this `BDuyZJVZpz4xyCt8zQFn89EDk `? is it possible to store it in environment variable and use it.

Reply:
and is it always the same or we need to change that random series in the future?

Reply:
That's an auth key. That's the one we use to keep track of open source projects. It's probably not going to change unless someone who uses docker-android gets a paid Polyverse account at some point, and it's probably won't need to change here, but it's totally possible to put it in an environment variable.

Each auth key links to an organization on our end that gets it's own set of uniquely scrambled packages, and those packages get re-scrambled on a regular basis, so building an image on one day will get a different scramble than an image built on a different day, and all those scrambles will be different than packages built for a different organization. So the key can be consistent, and you can get re-scrambled binaries on a regular basis.

Reply:
I will merge this PR then, but I will move  / hide that auth key on travis build and pass it through environment variable. Thanks again for the PR.